wgsl/index.bs

```diff
@@ -10028,7 +10078,9 @@ textureSampleLevel(t: texture_external, s: sampler, coords: vec2<f32>) -> vec4<f
   The optional texel offset applied to the unnormalized texture coordinate
   before sampling the texture. This offset is applied before applying any
   texture wrapping modes.<br>
-  The `offset` expression must be a `const_expression` expression (e.g. `vec2<i32>(1, 2)`).<br>
+  The `offset` expression must be either:
+    * a `const_expression` expression (e.g. `vec2<i32>(1, 2)`).<br>
```

Comment:
This construction occurs multiple times.
Maybe define a term, e.g. creation-time value.


Also, it's kind of weird that we can't combine them in an expression.  I guess that would be a TODO since I think we haven't settled on what exactly should be constexpr( ?)